src/frontend/src/components/displayError.ts

```diff
@@ -31,7 +33,11 @@ ${options.detail}</pre
       })}
 
       <div class="l-stack">
-        <button id="displayErrorPrimary" class="c-button c-button--primary">
+        <button
+          id="displayErrorPrimary"
+          data-error-code=${ifDefined(options.errorCode)}
```

Comment:
Is this related to disabling the PIN?

Reply:
It's related to the tests for disabling the PIN; I wanted to check that the error message shown was indeed that PIN auth is not allowed